<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_identificationuser</class>
 <widget class="QDialog" name="dlg_identificationuser">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>245</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>245</height>
   </size>
  </property>
  <widget class="QLabel" name="IdUserlabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginlineEdit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MDPlabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="UpPushButton" name="OKpushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>Connecter</string>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/start.png</normaloff>:/start.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Meta+Return</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="MDPlineEdit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>150</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhLatinOnly|Qt::ImhLowercaseOnly|Qt::ImhUppercaseOnly</set>
   </property>
   <property name="maxLength">
    <number>12</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="UpPushButton" name="AnnulpushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="icon">
    <iconset resource="../assets/Images/images.qrc">
     <normaloff>:/Supprime.png</normaloff>:/Supprime.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="UpLabel" name="ConnectServerupLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion au serveur</string>
    </property>
   </widget>
   <widget class="UpLabel" name="ConnectBaseupLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vérification de  la base  patients</string>
    </property>
   </widget>
   <widget class="UpLabel" name="IconServerOKupLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>20</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/Images/images.qrc">:/unchecked.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="UpLabel" name="IconBaseOKupLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>50</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/Images/images.qrc">:/unchecked.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="UpLabel" name="ConnectUserupLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vérification utilisateur</string>
    </property>
   </widget>
   <widget class="UpLabel" name="IconUserOKupLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>80</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/Images/images.qrc">:/unchecked.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpLabel</class>
   <extends>QLabel</extends>
   <header>uplabel.h</header>
  </customwidget>
  <customwidget>
   <class>UpPushButton</class>
   <extends>QPushButton</extends>
   <header>uppushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LoginlineEdit</tabstop>
  <tabstop>MDPlineEdit</tabstop>
  <tabstop>OKpushButton</tabstop>
  <tabstop>AnnulpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../assets/Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
